### ISSUE TITLE
Great Project

### ISSUE THREAD

**Issue by joshuaji03** [whichsteven/UW_Point_Cloud#10]
I feel like a point cloud projection will be more and more useful under a world that is highly dependent on the internet. I was wondering whether the point cloud visualization is able to project an indoor view?

**Comment by lolakang** [comment]
You are right Joshua. I took a class about point cloud visualization last quarter, and my professor actually made one point cloud in the classroom. So I believe that we are able to project indoor.
